src/components/Landing/styles.ts

```diff
@@ -2,4 +2,8 @@ import styled from 'styled-components'
 
 export const Container = styled.div`
```

Comment:
As possibly we will not redesign this landing page for some time, could we add some top margin here at least? :)